src/app.js

```diff
@@ -156,15 +157,74 @@ var run = function () {
   chromeCloudSync()
 
   // ----------------- editor ----------------------
-
   var editor = new Editor()
 
-  // ----------------- tabbed menu -------------------
+  // -------------- FileExplorer -------------------
+  var sources = {
+    'test/client/credit.sol': '',
+    'src/voting.sol': '',
+    'src/leasing.sol': '',
+    'src/gmbh/contract.sol': false,
+    'src/gmbh/test.sol': false,
+    'src/gmbh/company.sol': false,
+    'src/gmbh/node_modules/ballot.sol': false,
+    'src/ug/finance.sol': false,
+    'app/solidity/mode.sol': true,
+    'app/ethereum/constitution.sol': true
+  }
+  Object.keys(sources).forEach(function (key) { files.set(key, sources[key]) })
+
+  var fileExplorerAPI = {
+    filesList: function () { return Object.keys(files.list()) },
+    filesRemove: function (path) {
+      [...window.files.querySelectorAll('.file .name')].forEach(function (span) {
```

Comment:
Can you put a note here that this removes tabs as well as files? Perhaps it would make sense pulling out the internal function of "close tab" and use it here as well as in close tab event.